bluetooth/src/iosMain/kotlin/scanner/Scanner.kt

```diff
@@ -62,12 +63,13 @@ actual class Scanner internal constructor(
             autoEnableSensors: Boolean,
             scanningStateRepo: ScanningStateFlowRepo,
         ): BaseScanner {
-            return Scanner(permissions, connectionSettings, autoRequestPermission, autoEnableSensors, scanningStateRepo)
+            return Scanner(permissions, connectionSettings, scanSettings, autoRequestPermission, autoEnableSensors, scanningStateRepo,)
         }
     }
 
     companion object {
         private const val TAG = "IOS Bluetooth Scanner"
+        private val defaultScanSettings: Map<Any?, *>? = null
```

Comment:
Default value for iOS behaves different that default values we pass for Android. iOS will by default report each discovered device once and Android on each new device emission.